assembl/static2/js/app/components/administration/exportSection.jsx

```diff
@@ -0,0 +1,161 @@
+// @flow
```

Comment:
It should be a new section in admininistration > voteSession > export instead of administration > export. No?

Reply:
I agree

Reply:
Why having two components doing the same thing ? This component as in adminstration/survey and I moved it one folder up and use it for both survey and voteSession

Reply:
yes, I wrote this comment before I saw that it was in fact a refactoring